templates/cisco_ios_show_lldp_neighbors.textfsm

```diff
@@ -1,5 +1,6 @@
-Value Required NEIGHBOR (\S{0,20})
+Value Required CAPABILITIES (\S+)
```

Comment:
Should this be a List?

Reply:
A list seems like a good idea to me.

Reply:
So I can get it to pretend to be a list, but do not understand TextFSM well enough to get this fully into a _real_ list.  Perhaps someone else can finish this off.

```
Value List CAPABILITIES ([A-Z,]+)
Value Required LOCAL_INTERFACE (\S+)
Value Required NEIGHBOR (\S{0,20})
Value Required NEIGHBOR_INTERFACE (\S+)

Start
  ^Device.*ID -> LLDP
  # Capture time-stamp if vty line has command time-stamping turned on
  ^Load\s+for\s+
  ^Time\s+source\s+is

LLDP
  ^${NEIGHBOR}.*\s${LOCAL_INTERFACE}\s+\d+\s+${CAPABILITIES}\s+${NEIGHBOR_INTERFACE} -> Record
  ^${NEIGHBOR}${LOCAL_INTERFACE}\s+\d+\s+${CAPABILITIES}\s+${NEIGHBOR_INTERFACE} -> Record
  ^\s+${LOCAL_INTERFACE}\s+\d+\s+${CAPABILITIES}\s+${NEIGHBOR_INTERFACE} -> Record
```
This produces the following output:
```
        {
            "capabilities": [
                "B,W,R"
            ],
            "local_interface": "Fa1/0/2",
            "neighbor": "Random AP",
            "neighbor_interface": "XXXX.YYYY.ZZZZ"
        }

```
Whereas the ideal output would be:
```
        {
            "capabilities": [
                "B",
                "W",
                "R"
            ],
            "local_interface": "Fa1/0/2",
            "neighbor": "Random AP",
            "neighbor_interface": "XXXX.YYYY.ZZZZ"
        }

```

***However***, it would still be great if this could be committed since something is better than nothing.

Reply:
I should note that for my sake the original pull request is sufficient because I can post-process the string just fine.  I've taken the list idea about as far as I can.

Reply:
The PR is still missing updating the test files with the new capture group, which is causing Travis to fail. The test files are in `tests/cisco_ios/show_lldp_neighbors/`.

Reply:
Thanks!  I believe I'm updated them appropriately, however I'm still receiving this error.  Any tips?
![travis-error](https://user-images.githubusercontent.com/2447655/70267806-cd7e5b00-1764-11ea-8307-c1cb4fb1cc85.png)